sphinx/source/governance.rst

```diff
@@ -123,4 +118,54 @@ Once the proposal has been accepted, nodes running the new code are authorised j
 
 .. note:: It is important to keep the code compatible with the previous version, since there will be a point in time in which the new code is running on at least one node, while the other version is running on a different node.
 
-.. note:: The safest way to restart or replace nodes is by stopping a single node running the old version and starting a node running the new version as a sequence of operations, in order to avoid a situation in which most nodes have been stopped, and new nodes will not be able to join since it would be impossible to reach a majority of nodes agreeing to accept new nodes (this restriction is imposed by the consensus algorithm).
\ No newline at end of file
+.. note:: The safest way to restart or replace nodes is by stopping a single node running the old version and starting a node running the new version as a sequence of operations, in order to avoid a situation in which most nodes have been stopped, and new nodes will not be able to join since it would be impossible to reach a majority of nodes agreeing to accept new nodes (this restriction is imposed by the consensus algorithm).
+
+
+Models
+------
+
+The entities operating a CCF network do not necessarily overlap with the members of that network. Although the scriptability of the governance model effectively allows a large number of possible arrangements, the following two schemes seem most likely:
+
+Non-member operators
+````````````````````
+
+It is possible for a set of operators to host a CCF network without being members. These operators could:
+
+- Start the network
+- Hand it over to the members for them to Open (TR, Section IV B)
```

Comment:
We should reference other sections of this doc here, rather than the TR.

Reply:
Ok